searchcore/src/vespa/searchcore/proton/server/resource_usage_notifier.h

```diff
@@ -54,32 +54,32 @@ class DiskMemUsageNotifier : public IDiskMemUsageNotifier {
     uint64_t                     _diskUsedSizeBytes;
     TransientResourceUsage       _transient_usage;
     Config                       _config;
-    DiskMemUsageState            _dmstate;
+    ResourceUsageState            _dmstate;
```

Comment:
Nit: indent off by one. Maybe it should also be renamed to `_usage_state` (or similar) since it's now less obvious what the "dm" prefix stands for.